bentoml/pytorch_lightning.py

```diff
@@ -54,13 +64,11 @@ def load(
         import bentoml.pytorch_lightning
         booster = bentoml.pytorch_lightning.load(
             'lit_classifier:20201012_DE43A2', device_id="cuda:0")
-    """  # noqa
-    model = model_store.get(tag)
-    weight_file = model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
-    _load: t.Callable[[str], "pl.LightningModule"] = functools.partial(
-        torch.jit.load, map_location=device_id
-    )
-    return _load(weight_file)
+    """  # noqa: LN001
+    bentoml_model = model_store.get(tag)
+    weight_file = bentoml_model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
+    model: "pl.LightningModule" = torch.jit.load(weight_file, map_location=device_id)  # type: ignore[reportPrivateImportUsage] # noqa: LN001
+    return model
```

Comment:
@parano here I make the name different because we have both BentoML Model and pytorch model